src/Cake.Core/Diagnostics/Console/ConsoleColorEx.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Cake.Core.Diagnostics.Console
+{
+    internal static class ConsoleColorEx
+    {
+        public const ConsoleColor Default = (ConsoleColor)(-1);
```

Comment:
I considered a couple of other options including changing references to `ConsoleColor` to `Nullable<ConsoleColor>` (and have `null` to indicate default) as well as creating a separate `CakeConsoleColor` enum with all elements of `System.ConsoleColor` + a default option, but both approaches would introduce a binary breaking change to the public API, including `IConsole` and friends, which I don't think is desirable for the v1 timeframe...

Reply:
I'd add this to Constants https://github.com/cake-build/cake/blob/develop/src/Cake.Core/Constants.cs

Reply:
Good idea. I moved it to `Cake.Core/Constants.cs`